__tests__/app.test.js

```diff
@@ -303,4 +303,51 @@ describe('POST /api/articles/:article_id/comments', () => {
       expect(body.msg).toBe('Invalid input');
     })
   })
-})
\ No newline at end of file
+})
+
+describe("GET /api/articles (queries)", () => {
+  it("200: responds with a body of articles sorted by [column]", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy('author', {
+          descending: true
+        });
+      });
+  });
+  it("200: responds with a body of articles in descending order (default)", () => {
+    return request(app)
+      .get("/api/articles?order=desc")
```

Comment:
if you're checking default descending order you need to omit the order query so you know it's happening even if no one sends the order through -> `.get("/api/articles")`